src/app/@ansyn/imagery/model/base-imagery-map.ts

```diff
@@ -106,7 +106,7 @@ export abstract class BaseImageryMap<T = any> {
 	}
 
 	fitToExtent(extent: any): Observable<any> {
-		throw new Error('Method not implemented.');
```

Comment:
Isn't it better to implement this function inside openlayers-disabled-map?

Reply:
we also have cesium-map(and maybe more in the future),
I think that is better to implement it here.
and map that want to change this behavior, will implement this function.

Reply:
Ok